src/app/components/core/configuration-manager/configuration-group/configuration-group.component.html

```diff
@@ -9,8 +9,8 @@
     </ul>
   </nav>
   <div class="configuration_group_wrapper">
-    <nav id="group_navigation" class="groupNav dragscroll mouse-scroll" role="tablist">
-      <div id="groupNavContents" class="nav" [ngClass]="{'groupNav_contents': !sourceName}">
+    <nav [id]="'group_navigation_'+from + '_' + sourceName" class="groupNav dragscroll mouse-scroll" role="tablist">
+      <div [id]="'groupNavContents_'+from + '_' + sourceName" class="nav">
```

Comment:
We have nav before as tag and here as a class?

Reply:
No issue in that. It is same as before. I only removed custom class i.e. `groupNav_contents` because property (which was being applied in class `groupNav_contents`) needs to be applied on for all.